cmd/zpool/zpool_main.c

```diff
@@ -2389,6 +2392,9 @@ zpool_do_import(int argc, char **argv)
 				mntopts = optarg;
 			}
 			break;
+		case 'p':
+			flags |= ZFS_IMPORT_SCRUB_PAUSE;
+			break;
```

Comment:
We've had cases in the past where this would have been useful.  For example issue #3658, where due to some unfortunate hardware/drivers the pool couldn't be imported and the scrub canceled quickly enough to avoid other problems.  This might also be a useful feature to aid with pool recovery scenarios.